rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -220,8 +220,8 @@ def test_migration_4(rotkehlchen_api_server):
                     assert web3_node.weight == FVal(node['weight'])
                     continue
         assert len(web3_nodes) >= 5
-        assert web3_nodes[5].node_info.owned is True
-        assert web3_nodes[5].node_info.endpoint == 'https://localhost:5222'
+        assert web3_nodes[4].node_info.owned is True
```

Comment:
Because the default nodes that we have are actually loaded via the 4th migration and when queried from the DB are sorted in alphabetical order by name. Since I changed the nodes, the test also needed an adjustment